src/ai2_internal/api.py

```diff
@@ -158,3 +158,9 @@ def to_mmda(self) -> mmda_ann.SpanGroup:
             box_group=self.box_group.to_mmda()if self.box_group else None,
             id=self.id
         )
+
+
+class Relation(BaseModel):
+    from_id: int
+    to_id: int
+    attributes: Attributes = Attributes()
```

Comment:
The thought is that one day the kind of relation will need to be specified here, as an attribute? But for now we leave it unspecified?

Reply:
More general than that -- any metadata associated with the relation. Scores, or whatever. Arbitrary bag of data.

Kyle has it in mind that when one annotates an MMDA doc with relations you would specify their identifier, which is a 3-tuple of `(type, sourceAnnoName, targetAnnoName)` -- but that identifier is not in the `Relation` object itself.